### ISSUE TITLE
Notify team on new release of pubid-iso

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/pubid-iso#207]
From @opoudjis:

please contact @andrew2net to arrange release of pubid-iso, and notify me when done, so I can merge the metanorma-iso PR. This will again need to be a major version release (0.4.0), and I note that the removal of the `with_date` option is now crashing relaton-iso.

_Originally posted by @opoudjis in https://github.com/metanorma/metanorma-iso/issues/657#issuecomment-1412058765_

**Comment by mico** [comment]
@andrew2net I published new release https://github.com/metanorma/pubid-iso/releases/tag/v0.4.0
We can't use `Idenfitier.new` anymore to create new identifier, it should be:
`Identifier.create` (see updated README https://github.com/metanorma/pubid-iso#usage)

**Comment by opoudjis** [comment]
Cool, waiting on Andrej.

**Comment by andrew2net** [comment]
@mico the new version of pubid-iso fails to parse "ISO/IEC 23008-1/WD Amd 1" identifier of the doc https://www.iso.org/standard/82727.html
```ruby
Pubid::Iso::Identifier.parse("ISO/IEC 23008-1/WD Amd 1")
TypeError: can't create instance of singleton class
from /Users/andrej/.rvm/gems/ruby-3.0.0/gems/pubid-iso-0.4.0/lib/pubid/iso/identifier.rb:20:in `new'
```
The previus version of this gem parses the ID without error.

**Comment by andrew2net** [comment]
@ronaldtse relaton-iso, relaton, and relton-cli v 1.15.0 are released with pubis-iso v 0.4.0 dependency. Can we close this issue?

**Comment by ronaldtse** [comment]
Thanks!
